src/Kestrel.Core/Internal/Http2/Http2FrameWriter.cs

```diff
@@ -260,7 +262,7 @@ public Task WriteDataAsync(int streamId, StreamOutputFlowControl flowControl, Re
             |                           Padding (*)                       ...
             +---------------------------------------------------------------+
         */
-        private Task WriteDataUnsynchronizedAsync(int streamId, ReadOnlySequence<byte> data, bool endStream)
+        private Task WriteDataUnsynchronizedAsync(int streamId, MinDataRate minRate, ReadOnlySequence<byte> data, long dataLength, bool endStream)
```

Comment:
Looks like in all instances where this is called, dataLength is equal to data.Length?

Reply:
True, but data.Length on a ReadOnlySequence iterates, so you might as well cache the result.

Reply:
I see